platform/ios/scripts/release-notes-jazzy.md.ejs

```diff
@@ -4,6 +4,17 @@ The Mapbox Maps SDK for iOS is an open-source framework for embedding interactiv
 
 ![Mapbox Maps SDK for iOS screenshots](img/screenshot.png)
 
-For setup information, check out the [Mapbox Maps SDK for iOS homepage](https://www.mapbox.com/ios-sdk/). For detailed usage instructions, read “[First steps with the Mapbox Maps SDK for iOS](https://www.mapbox.com/help/first-steps-ios-sdk/)” and consult the [online examples](https://www.mapbox.com/ios-sdk/examples/). A [full changelog](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/CHANGELOG.md) is also available.
+For setup information, check out the [Mapbox Maps SDK for iOS homepage](https://www.mapbox.com/ios-sdk/). For detailed usage instructions, read “[First steps with the Mapbox Maps SDK for iOS](https://www.mapbox.com/help/first-steps-ios-sdk/)” and consult the [online examples](https://www.mapbox.com/ios-sdk/examples/).
 
 If you have any questions, please see [our help page](https://www.mapbox.com/help/). We welcome your [bug reports, feature requests, and contributions](https://github.com/mapbox/mapbox-gl-native/issues/).
+
+## Changes in version <%-CURRENTVERSION%>
+
+<%-CURRENTCHANGELOG-%>
+
+<% OTHERRELEASES.forEach(function(release) { %>
+## <%-release.version%>
```

Comment:
The version number on its own looks a little bare. How about prepending “Changes in version” like in the heading above?

Reply:
I dunno, I’m not entirely happy either way — this seems redundant. 🤷‍♂️

![mapbox maps sdk for ios reference 20180821](https://user-images.githubusercontent.com/1198851/44436940-c2121b00-a585-11e8-8975-0fbb2752cf27.jpg)

Reply:
Perhaps a subtle `<hr>` somewhere?

Reply:
How about “Recent changes” as h2, “Changes in version …” as h3, and the categories as h4?

Reply:
> How about “Recent changes” as h2, “Changes in version …” as h3, and the categories as h4?

![mapbox maps sdk for ios reference 2](https://user-images.githubusercontent.com/1198851/44477752-8ae54d80-a609-11e8-9bcc-80931e2de64d.jpg)

... yeah, “Changes in version” still feels redundant.

Reply:
Here’s without, which is my preference:

![mapbox maps sdk for ios reference 20180822](https://user-images.githubusercontent.com/1198851/44477787-ae0ffd00-a609-11e8-9d35-23966f797583.jpg)

/cc @colleenmcginnis

Reply:
Went with bare version numbers in 5570fabef938c9bfee8f04e348f5ae4f0e9d5a6c.